### PR TITLE
Replaced GraphiQL with GraphiQLWithExtensions

### PR DESCRIPTION
**Pull request by imolorhe**
This replaces the default GraphiQL
with one that includes GraphiQL explorer

Addresses #451

### CODE REVIEW

**Comment by sungam3r** [comment]
So simple?!

File: src/Ui.GraphiQL/Internal/graphiql.cshtml [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	  favored resource bundler.
 	 -->
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/graphiql/0.13.0/graphiql.min.css" />
```
Comment (sungam3r) [inline src/Ui.GraphiQL/Internal/graphiql.cshtml:42]: 1.0.6 ?

File: src/Ui.GraphiQL/Internal/graphiql.cshtml [hunk 2]
```diff
@@ -40,7 +40,7 @@
...
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/graphiql/0.13.0/graphiql.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.min.js"></script>
 
 	<script src="https://unpkg.com/subscriptions-transport-ws@0.8.3/browser/client.js"></script>
```
Comment (sungam3r) [inline src/Ui.GraphiQL/Internal/graphiql.cshtml:45]: 0.9.18 ?

**Comment by sungam3r** [comment]
Thanks @imolorhe , it works! I just have one question about bumping versions for two js packages. Is it worth doing this? We haven't changed this cshtml for a long time.

**Comment by imolorhe** [comment]
> So simple?!

😄 Ideally it's meant to be more complicated but the folks that built the GraphiQL explorer anticipated this kind of use case and created an already bundled version that is just a drop-in replacement for GraphiQL.

**Comment by varasinp** [comment]
I didn't realize that Onegraph people already created that "graphiqlWithExtensions.js:" 
Thank you!!

I also wonder whether or not we should upgrade the version number for the two js packages like @sungam3r  mentioned.

**Comment by imolorhe** [comment]
I've update the package versions and tested them

**Review by sungam3r — APPROVED** [review]

**Pull request merged**